virtcontainers/qemu_amd64.go

```diff
@@ -73,7 +74,7 @@ var supportedQemuMachines = []govmmQemu.Machine{
 
 // returns the maximum number of vCPUs supported
 func maxQemuVCPUs() uint32 {
-	return uint32(240)
+	return uint32(runtime.NumCPU())
```

Comment:
There will be some problems if the physical machine has more than 240 physical CPUs, right?
I think we should choose `MAX(runtime.NumCPU(), 240)` here